### PR TITLE
UN-175

### PR DESCRIPTION
**Pull request by JohnHeeryTNA**
Related ticket(s):
-  https://national-archives.atlassian.net/browse/UN-175

This is the combined BE and FE work for the home page tidy up

## How to check these changes

Please review the code via Github a local. build is available on my laptop

## Before assigning to reviewer, please make sure you have

- [x] Checked things thoroughly myself before handing over to reviewer.
- [ ] Included the ticket number in the PR title to help us keep track of changes.
- [x] Ensured that PR includes only commits relevant to the ticket.
- [x] Waited for all CI jobs to pass before requesting a review.
- [ ] Added/updated tests and documentation where relevant.

## For Reviewer

Once PR is merged, check and arrange to deploy on platform-sh.

### CODE REVIEW

**Comment by sdwilkes** [comment]
HTML and CSS all looks good, but I'm going to suggest @TNA-Allan checks the Python, so I will reassign it to him. Thanks.

File: etna/core/blocks/cta.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        default="Browse highlights of the collection through topics including:",
+    )
+    page = blocks.PageChooserBlock(page_type="collections.TopicExplorerIndexPage")
+
+    class Meta:
+        template = "collections/blocks/topic_explorer_new.html"
+        help_text = "Outputs all topic child pages"
+        icon = "th-large"
+        help_text = "Outputs all topic child pages"
+        icon = "th-large"
```
Comment (TNA-Allan) [inline etna/core/blocks/cta.py:43]: Repetitive variables assignment - `help_text` and `icon`

File: etna/core/blocks/cta.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+#    TimePeriodExplorerIndexBlock,
+#    TopicExplorerIndexBlock,
+# )
+#
+from etna.core.blocks import (  # ContentImageBlock,
+    PageListBlock,
+    SectionDepthAwareStructBlock,
+)
+
+# from wagtail.images.blocks import ImageChooserBlock
```
Comment (TNA-Allan) [inline etna/core/blocks/cta.py:13]: Dead code commented. Is this required?

File: etna/core/blocks/cta.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+class TopicExplorerBlock(blocks.StructBlock):
+    heading = blocks.CharBlock(max_length=100, default="Explore by topic")
+    sub_heading = blocks.CharBlock(
+        max_length=200,
+        default="Browse highlights of the collection through topics including:",
+    )
+    page = blocks.PageChooserBlock(page_type="collections.TopicExplorerIndexPage")
+
+    class Meta:
+        template = "collections/blocks/topic_explorer_new.html"
```
Comment (TNA-Allan) [inline etna/core/blocks/cta.py:39]: 1. Don't know how long a filename will stay `_new` or how to differentiate it from one without `_new`; could we suffix other than `_new`?
2. After adding `Topic Explorer` in for root page - home page, was not able to see `topic_explorer_new.html` in Debug Tool Bar in Preview. Is there any other way to view it?

File: etna/core/blocks/cta.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+class TimePeriodBlock(blocks.StructBlock):
+    heading = blocks.CharBlock(max_length=100, default="Explore by time period")
+    sub_heading = blocks.CharBlock(
+        max_length=200,
+        default="Discover 1,000 years of British history through time periods including:",
+    )
+    page = blocks.PageChooserBlock(page_type="collections.TimePeriodExplorerIndexPage")
+
+    class Meta:
+        template = "collections/blocks/time_period_explorer_new.html"
```
Comment (TNA-Allan) [inline etna/core/blocks/cta.py:25]: 1. Don't know how long a filename will stay `_new` or how to differentiate it from one without `_new`; could we suffix other than `_new`?
2. After adding `Time Period` in for root page - home page, was not able to see `time_period_explorer_new.html` in Debug Tool Bar in Preview. Is there any other way to view it?

**Review by TNA-Allan — CHANGES_REQUESTED** [review]
1. We could use branch name next time like mentioned in https://nationalarchives.github.io/ds-wagtail/developer-guide/project-conventions/ ex: feature/UN-175-home-page - perhaps with ticket number hyphened though.
2.  What is the full form of `cta`?  Could we use a bit verbose name over here `etna/core/blocks/cta.py`?

**Comment by LeanneEmans** [comment]
Hi @mozaki1 and @TNA-Allan  and @jamesbiggs 

I believe that Mo originally wrote the Python code. Mo please can you review the comments Allan has made above please?

File: etna/home/migrations/0011_add_feature_insights.py [hunk 5]
```diff
@@ -0,0 +1,138 @@
+# Generated by Django 3.2.14 on 2022-08-02 21:07
```
Comment (TNA-Allan) [inline etna/home/migrations/0011_add_feature_insights.py:1]: As discussed, multiple migrations with 0011xxx : previous fil[e 0011_add_search_image_fields.py](
https://github.com/nationalarchives/ds-wagtail/blob/develop/etna/home/migrations/0011_add_search_image_fields.py) file _etna/home/migrations/0011_add_feature_insights.py_ both ) 0011, and we want to have one single file for migrations 0011, 0012, 0013 changes in this ticket. 
Example  0012_homepage_featured_collections_and_more.py

File: etna/home/migrations/0012_merge_20220926_1451.py [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# Generated by Django 4.0.7 on 2022-09-26 14:51
```
Comment (TNA-Allan) [inline etna/home/migrations/0012_merge_20220926_1451.py:1]: same as comment for 0011_add_feature_insights.py

File: etna/home/migrations/0013_alter_homepage_body.py [hunk 7]
```diff
@@ -0,0 +1,124 @@
+# Generated by Django 4.0.7 on 2022-09-30 12:47
```
Comment (TNA-Allan) [inline etna/home/migrations/0013_alter_homepage_body.py:1]: same as comment for 0011_add_feature_insights.py

**Review by TNA-Allan — CHANGES_REQUESTED** [review]

File: etna/core/blocks/cta.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (mozaki1) [reply etna/core/blocks/cta.py]: The name was here always and not part of this work.

File: etna/home/migrations/0011_add_feature_insights.py (hunk 5 above)
Comment (TNA-Allan) [reply etna/home/migrations/0011_add_feature_insights.py]: Got an error `django.db.utils.ProgrammingError: column "search_image_id" of relation "home_homepage" already exists`
Fix:
Restore `0011_add_search_image_fields.py`
Delete `0012xx`
Get Staging Data and run new migrations which creates new  `0012_homepage_featured_collections_and_more.py`

File: templates/insights/blocks/featured_collection_new.html [hunk 8]
```diff
@@ -0,0 +1,13 @@
+{% load wagtailimages_tags %}
```
Comment (TNA-Allan) [inline templates/insights/blocks/featured_collection_new.html:1]: templates/insights/blocks/featured_collection_new.html looks unused. Please check if required.

**Review by TNA-Allan — CHANGES_REQUESTED** [review]

**Comment by TNA-Allan** [comment]
Reviewed last feedback. Looks good. Passing to FE as per notes 31/10 on ticket.

**Review by TNA-Allan — APPROVED** [review]
Looks Good. Thanks

**Pull request merged**